runtime/Cargo.toml

```diff
@@ -60,6 +60,8 @@ pallet-transaction-payment = { git = "https://github.com/paritytech/substrate",
 pallet-transaction-payment-rpc-runtime-api = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.17", default-features = false }
 pallet-authorship = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.17", default-features = false }
 pallet-session = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.17", default-features = false }
+pallet-society = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.17", default-features = false }
+pallet-collective = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.17", default-features = false }
```

Comment:
Do we need this dependency?

Reply:
No, we don't require this dependency but  I am not able to build node without adding them.  I have asked the same  question in  [Issue](https://github.com/paritytech/substrate/issues/10548 )

Reply:
Removed those dependencies. runtime-benchmarks feature flag was needed to be enabled for polkadot-cli  in ./node/cargo.toml

for details regarding the issue :  [Issue Link](https://substrate.stackexchange.com/questions/240/missing-successful-origin-in-implementation-when-running-runtime-benchmarks)